### ISSUE TITLE
[AasxPluginDocumentShelf] Error: AasxPluginDocumentShelf / converting previews: Entries cannot be opened multiple times in Update mode. Entries cannot be opened multiple times in Update mode. at    at System.IO.Compression.ZipArchiveEntry.OpenInUpdateMode()

### ISSUE THREAD

**Issue by HBUDE** [eclipse-aaspe/aaspe#181]
Hello,

I don't know if it's a bug or simply wrong usage, but when I add a submodel 'Documentation' via template (Create->new submodel from plugin->AAS plugin document shelf | Document (recommended version) ), and after adding some documents, I get the following error message:

[AasxPluginDocumentShelf] Error: AasxPluginDocumentShelf / converting previews: Entries cannot be opened multiple times in Update mode. Entries cannot be opened multiple times in Update mode. at    at System.IO.Compression.ZipArchiveEntry.OpenInUpdateMode()...

Any idea whats wrong ?!

If you need additional information, please tell.

Thanks, bye
Harald

**Comment by juileetikekar** [comment]
Hi @HBUDE,

Thank you for raising the issue!!

The issue was reproduced successfully. We will keep you informed with further updates.

**Comment by MichaelHoffmeisterFesto** [comment]
Yes, I know about this bug. It is associated with some concurrent thread handling and the mysterious OPC/ZIP library and will take some time to be diagnosed and fix,
It is annoying me, as well.
Please leave this issue open.
